Classes/Service/NodeRedirectService.php

```diff
@@ -282,7 +281,11 @@ protected function createUriPathsAcrossDimensionsForNode(string $nodeIdentifier,
                 continue;
             }
 
-            $nodeUriPath = $this->buildUriPathForNode($nodeInDimensions);
+            try {
+                $nodeUriPath = $this->buildUriPathForNode($nodeInDimensions);
+            } catch (\Exception $_) {
+                continue;
```

Comment:
since we silently skip failures in line 281 already, I chose not to log the exception in this case too – but this could be argued of course